packages/app/src/systems/Error/pages/ReportErrors/ReportErrors.tsx

```diff
@@ -0,0 +1,123 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Icon,
+  Stack,
+  FuelLogo,
+  Heading,
+  Button,
+  Box,
+  Text,
+  Input,
+} from '@fuel-ui/react';
+
+import { useReportError } from '../../hooks';
+
+import { WALLET_WIDTH, WALLET_HEIGHT } from '~/config';
+import { Layout } from '~/systems/Core';
+import { coreStyles } from '~/systems/Core/styles';
+
+export function ReportErrors() {
+  const { handlers, isLoadingDontSend, isLoadingSendOnce, errors } =
+    useReportError();
+
+  return (
+    <Layout title="Error" isPublic>
+      <Stack>
+        <Layout.Content as="div">
+          <Box as="div">
+            <Stack align="center">
+              <FuelLogo size={60} />
+              <Heading as="h3">Help us improve Fuel Wallet</Heading>
+            </Stack>
+            <Stack align="center">
+              <Icon icon="AlertTriangle" size={32} />
+              <Heading as="h5">What happened?</Heading>
+              <Text>
+                Fuel Wallet has detected unreported errors / crashes. We&apos;re
+                sorry for the inconvenience.
+                <br />
+                <br />
+                Would you like to send us the error report to help us improve
+                Fuel Wallet?
+              </Text>
+
+              <Input isDisabled={true} css={styles.textArea}>
+                <Input.Field as="textarea" value={errors} />
+              </Input>
+            </Stack>
+          </Box>
+        </Layout.Content>
+        <Layout.BottomBar>
+          <Stack css={styles.fullWidth}>
+            <Button
+              color="accent"
+              variant="ghost"
+              isDisabled={isLoadingSendOnce}
+              isLoading={isLoadingSendOnce}
+              onPress={handlers.reportErrorsOnce}
+              aria-label="Report Error"
+            >
+              Send to Fuel
+            </Button>
+            <Button
+              color="gray"
+              isDisabled={isLoadingDontSend}
+              isLoading={isLoadingDontSend}
+              onPress={handlers.dontReportErrors}
+              aria-label="Don't send error report"
+            >
+              Don&apos;t Send
```

Comment:
Ignore would probably be a better choice of wording here, don't you think?
```suggestion
              Ignore
```